python/core/auto_generated/raster/qgsrasterdataprovidertemporalcapabilities.sip.in

```diff
@@ -140,6 +140,20 @@ datetime values for the provider.
 .. seealso:: :py:func:`defaultInterval`
 
 .. versionadded:: 3.20
+%End
+
+    void setRequestedTemporalRange( const QgsDateTimeRange &range );
```

Comment:
Can we skip this from the sip bindings? I'm still not 100% on this particular bit of api, and don't want it locked into stable api yet